internal/access/organization.go

```diff
@@ -41,13 +41,13 @@ func GetOrganization(c *gin.Context, id uid.ID) (*models.Organization, error) {
 	return data.GetOrganization(rCtx.DBTxn, data.GetOrganizationOptions{ByID: id})
 }
 
-func CreateOrganization(c *gin.Context, org *models.Organization) error {
+func CreateOrganization(c *gin.Context, org *models.Organization, google *models.Provider) error {
```

Comment:
How do you plan on scaling this to other global IDPs, e.g. GitHub?

Reply:
We dont have plans for those at the moment